### PR TITLE
added venue information

### PR DESCRIPTION
**Pull request by mfc**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -61,7 +61,28 @@ Venue
...
+
+---
+
+<h1 align="left" style="margin-top: 0px;">
+Hygiene and Safety Guidelines
+</h1>
+
+<h3 align="left" style="margin-top: 0px;">
+
+The event space is a 2G+ space, please familiarize yourself with the <a href="https://co-up.de/hygiene-safety-guidelines/" target="_blank">Hygiene and Safety Guidelines</a>. We will have masks and antigen tests available to event participants should you want one.
```
Comment (pietrushnic) [inline docs/index.md:85]: @mfc We agree to add information about self-assertion in registration form.

Comment (mfc) [reply docs/index.md:85]: yes, i don't have access to the registration form but feel free to add similar information there. i think it is nice to include this information on the summit website as well.

Comment (pietrushnic) [reply docs/index.md:85]: @mfc we will do that. So let's wait for mergining until @Asiderr will provide registriation form which we discussed yesterday.

Comment (Asiderr) [reply docs/index.md:85]: @mfc The link: https://cloud.3mdeb.com/index.php/apps/forms/Y7Mb7LHLmXWspY4o

**Pull request merged**